frontend/packages/git-service/src/services/github-service.ts

```diff
@@ -105,7 +112,7 @@ export class GithubService extends BaseService {
       const resp = await this.client.repos.getContents({
         owner: this.metadata.owner,
         repo: this.metadata.repoName,
-        path: 'Dockerfile',
+        path: `${this.metadata.contextDir}/Dockerfile`,
```

Comment:
Would this work with `/Dockerfile` as well because the path would become that if there's no contextDir?

Reply:
GitHub API also works, updated nock tests.